core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
+    """Checks if conditions for training are satisfied.
+
+    Args:
+        state: Domain object for the State.
+
+    Returns:
+        bool: True, if the conditions are satisfied.
+    """
+    training_examples = [
+        [doc, []] for doc in state.interaction.confirmed_unclassified_answers]
+    labels_ctr = 0
+    for (answer_group_index, answer_group) in enumerate(
+            state.interaction.answer_groups):
+        classifier_rule_spec_index = answer_group.get_classifier_rule_index()
+        if classifier_rule_spec_index is not None:
+            classifier_rule_spec = answer_group.rule_specs[
+                classifier_rule_spec_index]
+        else:
+            classifier_rule_spec = None
+        if classifier_rule_spec is not None:
```

Comment:
I kept this clause because this I use it to get the number of labels in the state. The number of times an answer group has a classifier_rule_spec (as Not None) means we have training data and also that we have one label response.